### ISSUE TITLE
Empty sitemap.xml

### ISSUE THREAD

**Issue by theodend** [Sterc/SEOTab#195]
Hello,
I've rtfm but my sitemap.xml stay empty. 
Only `<sitemapindex xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"> </sitemapindex>` for `[[!StercSeoSiteMap?&type=`index`]]`
Any idea ?
Thank you

**Comment by degoya** [comment]
Do you have multiple contexts? 

maybe you need to define them like this
`[[!StercSeoSiteMap?&contexts=`en,de`]]`

or try with the default context
`[[!StercSeoSiteMap?&contexts=`web`]]`

**Comment by crimsonpixel** [comment]
Hi there, I am experiencing this issue too. 

I have tried default setting but the sitemap continues to be empty. Template is empty and no richtext.

MODX 2.8.1
PHP 7.3.23

And latest version of SEO Pro and SEO Tab running on the site.

**Comment by wallacio** [comment]
Yep, empty site map for me too. Fresh install of SEOTab on an existing 2.7.3 site, PHP 7.3. I say empty, it does contain a reference to itself which isn't particularly useful. Single, default context of 'web', no options on the snippet.

```
<urlset>
  <url>
    <loc>https://site.com/sitemap.xml</loc>
    <lastmod>2020-11-30T09:51:57+00:00</lastmod>
    <changefreq>weekly</changefreq>
    <priority>0.5</priority>
  </url>
</urlset>
```

The error log contains a smoking gun, which has just started appearing since installing the extension:

[2020-11-30 09:42:57] (ERROR @ /home/int29er/public_html/core/xpdo/xpdo.class.php : 644) Could not load class: Babel from babel.
[2020-11-30 09:42:57] (ERROR @ /home/int29er/public_html/core/xpdo/xpdo.class.php : 1247) Problem getting service babel, instance of class Babel, from path /home/int29er/public_html/core/components/babel/model/babel/
